### PR TITLE
Created endpoints for Mosaic

### PR DESCRIPTION
**Pull request by LBHBReynolds-Carr**
- Created Boundary Response, Gateway and Domain for Mosaic information
- Corrected outstanding error: Academy gateway was creating boundary
response
- Extracted SetUpMessageHandlerToReturnJson method to static test helper
- All tests passing locally

### CODE REVIEW

File: ResidentInformationApi.Tests/V1/Helper/TestHelper.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+namespace ResidentInformationApi.Tests.V1.Helper
+{
+    public static class TestHelper
+    {
+        public static void SetUpMessageHandlerToReturnJson(Mock<HttpMessageHandler> messageHandler, string rqpString = null, string expectedJsonString = null)
+        {
+            if (expectedJsonString == null)
+            {
+                expectedJsonString = new Fixture().Create<String>();
```
Comment (elcorbs) [inline ResidentInformationApi.Tests/V1/Helper/TestHelper.cs]:
```suggestion
                expectedJsonString = new Fixture().Create<string>();
```

File: ResidentInformationApi.Tests/V1/Helper/TestHelper.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+using System.Web;
+using AutoFixture;
+using Moq;
+using Moq.Protected;
+
+namespace ResidentInformationApi.Tests.V1.Helper
+{
+    public static class TestHelper
+    {
+        public static void SetUpMessageHandlerToReturnJson(Mock<HttpMessageHandler> messageHandler, string rqpString = null, string expectedJsonString = null)
```
Comment (elcorbs) [inline ResidentInformationApi.Tests/V1/Helper/TestHelper.cs:15]: Nice refactoring!

**Review by elcorbs — APPROVED** [review]
Really good :)

File: ResidentInformationApi.Tests/V1/Helper/TestHelper.cs (hunk 2 above)
Comment (LBHBReynolds-Carr) [reply ResidentInformationApi.Tests/V1/Helper/TestHelper.cs:15]: Thanks, I was able to find some similar test helpers to base this off

**Pull request merged**